surah/models.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+'''This module contains Surahs of Holy Quran'''
+from django.db import models
+
+
+class Surah(models.Model):
+    '''Surah class with id name and number of ayahs'''
+    surah_id = models.AutoField(primary_key=True)
+    surah_name = models.CharField(max_length=250)
```

Comment:
1. Why 250? I don't remember so long names of surahs.
2. Write some help text.

Reply:
No much difference unless we go over 255 with another byte, don't know actually the longest name & wasn't sure about number of max chars.
descriptions is pain :(

Reply:
Each ascii symbol is 1 byte. Each utf-8 symbol consumes even more. I’m not completely sure how it's implemented in DBs, but it's a waste of resources.

0-255 is a range of values that you can encode with 1 byte. It's a different thing.

Reply:
Ok but it impacts on temporary files, so I agree to change it according lenght